Source/storm/storm.cpp

```diff
@@ -15,16 +16,16 @@
 #include "Radon.hpp"
 
 #include "DiabloUI/diabloui.h"
-#include "display.h"
 #include "dx.h"
 #include "options.h"
-#include "paths.h"
-#include "sdl_compat.h"
-#include "stubs.h"
+#include "utils/display.h"
+#include "utils/paths.h"
+#include "utils/sdl_compat.h"
+#include "utils/stubs.h"
 
 namespace devilution {
 
-DWORD nLastError = 0;
+uint32_t nLastError = 0;

```

Comment:
Unrelated?

Reply:
Much easier than to try and deal with the nested relation between storm.h and minwin.h. It was either this, have it not compile, or add a hack.

Reply:
... and it's gone.

https://github.com/diasurgical/devilutionX/commit/e1278cba7b611e6ba130ccbebcff40746e73f65a#diff-7501f2d1b1864ababb342df86d326f82d9f92f4bb34b56b8ed91ac75c87b1a0a